# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/nfs-utils/hotfix-gcc-4.2.patch.avr32
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- nfs-utils-2.4.3/utils/mount/stropts.c.vanilla	2021-10-07 12:31:46.430690497 +0200
+++ nfs-utils-2.4.3/utils/mount/stropts.c	2021-10-07 12:32:21.349692988 +0200
@@ -783,11 +783,11 @@
 			fmt = "vers=%lu.%lu";
 			break;
 		}
-#pragma GCC diagnostic ignored "-Wformat-nonliteral"
+//#pragma GCC diagnostic ignored "-Wformat-nonliteral"
 		snprintf(version_opt, sizeof(version_opt) - 1,
 			fmt, mi->version.major,
 			mi->version.minor);
-#pragma GCC diagnostic warning "-Wformat-nonliteral"
+//#pragma GCC diagnostic warning "-Wformat-nonliteral"
 
 		if (po_append(options, version_opt) == PO_FAILED) {
 			errno = EINVAL;
@@ -1094,9 +1094,9 @@
 		if (nfs_try_mount(mi))
 			return EX_SUCCESS;
 
-#pragma GCC diagnostic ignored "-Wdiscarded-qualifiers"
+//#pragma GCC diagnostic ignored "-Wdiscarded-qualifiers"
 		if (errno == EBUSY && is_mountpoint(mi->node)) {
-#pragma GCC diagnostic warning "-Wdiscarded-qualifiers"
+//#pragma GCC diagnostic warning "-Wdiscarded-qualifiers"
 			/*
 			 * EBUSY can happen when mounting a filesystem that
 			 * is already mounted or when the context= are
--- nfs-utils-2.4.3/utils/nfsstat/nfsstat.c.vanilla	2021-10-07 12:31:23.800688883 +0200
+++ nfs-utils-2.4.3/utils/nfsstat/nfsstat.c	2021-10-07 12:31:30.884689389 +0200
@@ -980,10 +980,10 @@
 			}
 			bufp = buf;
 			for (; curindex < numvals; curindex++) {
-#pragma GCC diagnostic ignored "-Wformat-nonliteral"
+//#pragma GCC diagnostic ignored "-Wformat-nonliteral"
 				n = sscanf(bufp, fmt, &ip->valptr[curindex],
 						&numconsumed);
-#pragma GCC diagnostic warning "-Wformat-nonliteral"
+//#pragma GCC diagnostic warning "-Wformat-nonliteral"
 				if (n != 1)
 					break;
 				if (is_proc) {
